docs/example1.html

```diff
@@ -0,0 +1,22 @@
+<head>
```

Comment:
The docs directory has `example1.html`, `basic-example.html`, and `github-pages-example-1.html`. I'm not sure the difference, or why the naming schemes are so different. Perhaps this should be named `github-pages-example-2.html`, and add this to `index.md`?